contracts/truefi/TrueLender.sol

```diff
@@ -430,9 +430,7 @@ contract TrueLender is ITrueLender, Ownable {
         uint256 denominator
     ) external override onlyPool {
         for (uint256 index = 0; index < _loans.length; index++) {
-            if (!_loans[index].transfer(recipient, numerator.mul(_loans[index].balanceOf(address(this))).div(denominator))) {
-                0;
-            }
+            require(_loans[index].transfer(recipient, numerator.mul(_loans[index].balanceOf(address(this))).div(denominator)));
```

Comment:
Is it possible to add `using SafeERC20 for ILoanToken` or cast `_loans[index]` as IERC20, then call `safeTransfer` instead of `require(...transfer(...))`? If not then this looks good to me.

Last week I tried calling `safeApprove()` for an ITrueFiPool, but unit tests that mocked out the pool wouldn't recognize `safeApprove()` as part of the ABI.